packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/PhaseFocusControls/PhaseFocusControls.js

```diff
@@ -23,9 +23,9 @@ function PhaseFocusControls(props) {
   return (
     <Box direction='column'  pad='none'>
       {data.map((series, seriesIndex) => {
-        const checked = phaseFocusedSeries === seriesIndex
-        const label = counterpart('VariableStarViewer.label', { id: seriesIndex + 1 })
         const { seriesOptions } = series
+        const checked = phaseFocusedSeries === seriesIndex
+        const label = seriesOptions.label || counterpart('VariableStarViewer.label', { id: seriesIndex + 1 })
```

Comment:
Do you need to check for `seriesOption`'s existence here? I noticed you checked for `series.seriesOptions?.label` in the container.